libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -319,6 +319,49 @@ object MapboxRouteLineUtils {
         return routeLineTrafficData
     }
 
+    @SuppressLint("LogNotTimber")
```

Comment:
We should remove `Timber` from `libnavui-maps`.

Reply:
Can you remove the dep from `libnavui-maps/build.gradle` altogether?

Reply:
We can follow up in a separate ticket. Could you cut a ticket when you have a chance @cafesilencio? 🙏